src/authentication/management/commands/get_admin_token.py

```diff
@@ -0,0 +1,14 @@
+from django.core.management.base import BaseCommand, CommandError
+from authentication.models import User
+
+
+class Command(BaseCommand):
+    help = "Prints the admin API token to stdout."
+
+    def handle(self, *args, **options):
+        admin_user = User.objects.get(username="admin")
```

Comment:
A couple questions:

- Do we want to hardcode "admin" here?
  - As a possible alternative, is there an "admin" role that's being used by User objects we could reference here instead, to see if there are any admin users that aren't named "admin" that might be being used?
- What if there are multiple tokens stored in the system for different users?
  - If there are multiple tokens, should this output all of them, or just the one for "admin"?

(Note that I know right now this is all overkill considering our current environment, but the potential here to code to our implementation instead of a more generic one makes me hesitant, especially since this is a public repo.)

Reply:
I just pushed an update to make this command more generic